packages/clerk-js/src/ui/styles/_alert.scss

```diff
@@ -1,13 +1,19 @@
-.error {
+.alert {
```

Comment:
🔧  In theory, this is a breaking change since the `cl-error` class was publicly targettable.

We need to make sure that the `cl-error` class still exists as a targetable class on Alert component

Am I missing something? cc @SokratisVidros  @igneel64

Reply:
Good catch @nikosdouvlis

Reply:
Anything in these styles can be considered public in a way so yes!

Reply:
@nikosdouvlis Only the `cl-error` class or the `Error` component as well?